backend/utils/crons/notification.py

```diff
@@ -0,0 +1,66 @@
+import pytz
```

Comment:
How frequently is this cronjojb been triggered?

Is being scheduled individually for each user?

Reply:
Every minute. No,  just one cron job runs for everyone.